### PR TITLE
Add twitter feed on homepage

### LINKED ISSUE #3280: Add Twitter section on the Homepage
Add twitter section on homepage between `Partner Organization` and `Cite Our Work`.

@Ram81 @RishabhJain2018

### PR DESCRIPTION
**Pull request by gautamjajoo**
Fixes #3280 . @Ram81 @RishabhJain2018 

Screenshot from 2021-03-03 12-14-45

### CODE REVIEW

**Review by scott1979rn — APPROVED** [review]

File: frontend_v2/src/app/components/home/home.component.ts [hunk 1]
```diff
@@ -50,6 +50,7 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
    */
   ngAfterViewInit() {
     this.SUBSCRIBE_FORM[this.subscribeForm] = this.components;
+    (<any>window).twttr.widgets.load();
```
Comment (Ram81) [inline frontend_v2/src/app/components/home/home.component.ts:53]: @gautamjajoo did you test what happens when `load()` call fails? If yes, how?

**Review by Ram81 — CHANGES_REQUESTED** [review]
@RishabhJain2018 where do we want to place the Twitter feed on the homepage?

File: frontend_v2/src/app/components/home/home.component.ts (hunk 1 above)
Comment (gautamjajoo) [reply frontend_v2/src/app/components/home/home.component.ts:53]: Actually we don't need `load()` without it the code would work

Comment (Ram81) [reply frontend_v2/src/app/components/home/home.component.ts:53]: @gautamjajoo please remove it

Comment (gautamjajoo) [reply frontend_v2/src/app/components/home/home.component.ts]: Done

**Comment by gautamjajoo** [comment]
@Ram81 We have a separate folder `twitter-feed, do we need that?

**Comment by Ram81** [comment]
We can remove it if it is not being used

**Comment by gautamjajoo** [comment]
@Ram81 Updated!

File: frontend_v2/src/app/components/home/home.component.scss [hunk 2]
```diff
@@ -5,6 +5,10 @@
   margin-right: auto;
 }
 
+.twitter-container {
+  margin-left:15vw;
```
Comment (Kajol-Kumari) [inline frontend_v2/src/app/components/home/home.component.scss:9]: using `vw` will make it misbehave for different browsers. Can we please use px instead?

File: frontend_v2/src/app/components/home/home.component.html [hunk 3]
```diff
@@ -385,8 +385,20 @@ <h3 class="fw-light center">Partner Organizations</h3>
         </div>
       </div>
     </section>
-    <!-- Arxiv Entry -->
+
```
Comment (Kajol-Kumari) [inline frontend_v2/src/app/components/home/home.component.html:388]: let's put `<!-- twitter container -->` as comment here inorder to be consistent

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@gautamjajoo please do the following changes -
1. The twitter container should be center aligned for small screens
2. Let's put some border /shadow around the iframe container inorder to improve it's look.

File: frontend_v2/src/app/components/home/home.component.scss (hunk 2 above)
Comment (gautamjajoo) [reply frontend_v2/src/app/components/home/home.component.scss]: Changed!

File: frontend_v2/src/app/components/home/home.component.html (hunk 3 above)
Comment (gautamjajoo) [reply frontend_v2/src/app/components/home/home.component.html:388]: Added the comment

**Comment by gautamjajoo** [comment]
Here's the updated screenshot:
Screenshot from 2021-08-05 23-49-26

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Comment by RishabhJain2018** [comment]
@gautamjajoo Can you please resolve the conflicts here?

**Pull request merged**